com.unity.cinemachine/Samples~/Input System/Split Screen Multiplayer/SimpleControls.inputactions

```diff
@@ -0,0 +1,241 @@
+{
+    "name": "SimpleControls",
+    "maps": [
+        {
+            "name": "gameplay",
```

Comment:
Why do you have a custom InutAcrtions asset for this scene?  Why not use the common one that's in shared assets?

Reply:
Fixed